README.md

```diff
@@ -46,7 +46,7 @@ Installation
 Add solidus_multi_domain to your Gemfile:
 
 ```ruby
-gem 'solidus_multi_domain'
+gem 'solidus_multi_domain', github: 'solidusio-contrib/solidus_multi_domain', branch: 'master'
```

Comment:
@afdev82 `branch: 'master'` shouldn't be needed and will be used by default. Can you remove that? Other than that, good to merge!